backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -165,70 +166,74 @@ export class AdminResolver {
 
   @Authorized([RIGHTS.CREATE_PENDING_CREATION])
   @Mutation(() => [Number])
-  async createPendingCreation(
-    @Args() { email, amount, memo, creationDate }: CreatePendingCreationArgs,
+  async adminCreateContribution(
+    @Args() { email, amount, memo, creationDate }: AdminCreateContributionArgs,
     @Ctx() context: Context,
   ): Promise<Decimal[]> {
+    logger.trace('adminCreateContribution...')
     const user = await dbUser.findOne({ email }, { withDeleted: true })
     if (!user) {
       throw new Error(`Could not find user with email: ${email}`)
     }
     if (user.deletedAt) {
-      throw new Error('This user was deleted. Cannot make a creation.')
+      throw new Error('This user was deleted. Cannot create a contribution.')
     }
     if (!user.emailChecked) {
-      throw new Error('Creation could not be saved, Email is not activated')
+      throw new Error('Contribution could not be saved, Email is not activated')
     }
     const moderator = getUser(context)
+    logger.trace('moderator: ', moderator.id)
     const creations = await getUserCreation(user.id)
+    logger.trace('creations', creations)
     const creationDateObj = new Date(creationDate)
     if (isCreationValid(creations, amount, creationDateObj)) {
-      const adminPendingCreation = AdminPendingCreation.create()
-      adminPendingCreation.userId = user.id
-      adminPendingCreation.amount = amount
-      adminPendingCreation.created = new Date()
-      adminPendingCreation.date = creationDateObj
-      adminPendingCreation.memo = memo
-      adminPendingCreation.moderator = moderator.id
-
-      await AdminPendingCreation.save(adminPendingCreation)
+      const contribution = Contribution.create()
+      contribution.userId = user.id
+      contribution.amount = amount
+      contribution.createdAt = new Date()
+      contribution.contributionDate = creationDateObj
+      contribution.memo = memo
+      contribution.moderatorId = moderator.id
+
+      logger.trace('contribution to save', contribution)
+      await Contribution.save(contribution)
     }
     return getUserCreation(user.id)
   }
 
   @Authorized([RIGHTS.CREATE_PENDING_CREATION])
-  @Mutation(() => CreatePendingCreations)
-  async createPendingCreations(
-    @Arg('pendingCreations', () => [CreatePendingCreationArgs])
-    pendingCreations: CreatePendingCreationArgs[],
+  @Mutation(() => AdminCreateContribution)
```

Comment:
please do not mix Rights/Rules with method names, especially when with replacement of the elopage process the user will capture his contributions himself and the admin will only confirm them